prow/jira/jira.go

```diff
@@ -782,3 +785,19 @@ func (l *retryableHTTPLogrusWrapper) Debug(msg string, context ...interface{}) {
 func (l *retryableHTTPLogrusWrapper) Warn(msg string, context ...interface{}) {
 	l.log.WithFields(l.fieldsForContext(context...)).Warn(msg)
 }
+
+// Search wraps SearchWithContext using the background context.
+func (jc *client) Search(jql string, options *jira.SearchOptions) ([]jira.Issue, *jira.Response, error) {
+	return jc.SearchWithContext(context.Background(), jql, options)
```

Comment:
What's the utility of this method? If we're adding *new* code, no well-formed caller should prefer `context.Background`, right?